tools/proofers/text_frame_std.inc

```diff
@@ -23,6 +23,7 @@ function echo_text_frame_std($ppage)
             "$code_url/scripts/text_validator.js",
             "$code_url/tools/proofers/proof_validate.js",
             "$code_url/tools/proofers/process_diacritcal_markup.js",
+            "$code_url/tools/proofers/dp_proof.js",
```

Comment:
Rather than including dp_proof.js here would it be better to access the one already loaded in tools/proofers/proof.php by prepending "top."? as is done for initializeStuff() referring to dp_scroll. Similarly elsewhere.

Reply:
Thanks @70ray - another new thing I've learned :)